### ISSUE TITLE
pgrouting failed to build on 10.10.1

### ISSUE THREAD

**Issue by dnix** [Homebrew/homebrew#35515]
https://gist.github.com/dnix/8c2b658b896d59143d80

Many thanks!

**Comment by dakcarto** [comment]
Hi @dnix. Looks like a Postgres in your Applications may be interfering:
```
-- POSTGRESQL_EXECUTABLE is /Applications/postgres.app/Contents/MacOS/Postgres
```
Homebrew's `postgresql` doesn't install a `postgres.app`, so there is probably incompatibilities in this line during the build (which appears to generate the error):
```
/usr/local/Library/ENV/4.3/clang    -DPGSQL_VERSION=94 -fPIC -O2 -g -bundle -Wl,-headerpad_max_install_names -bundle_loader /Applications/postgres.app/Contents/MacOS/Postgres -bundle -o ../../../lib/librouting_tsp.so CMakeFiles/routing_tsp.dir/tsp2.c.o CMakeFiles/routing_tsp.dir/tsplib.c.o 
```

Try moving the `postgres.app` out of Applications temporarily, or fix/comment where it is being added to PATH (in your bash profile?), then try and install `pgrouting`.

**Comment by mikemcquaid** [comment]
I've created a bottle in c8f86c2db816022ce4ad485e619b1ac7fd43faf3 which may help.

**Comment by dnix** [comment]
That did the trick. Removed postgress.app and `brew install pgrouting` worked as expected. Huge thanks to you both.

**Comment by sanak** [comment]
@mikemcquaid
Created bottle includes only the following files, and `brew list pgrouting` show nothing.
<pre>
2.0.0
  +- COPYING
  +- INSTALL_RECEIPT.json
  +- README.md
</pre>
So, to install pgrouting actually, `--build-from-source` was necessary.
Should I create this as a new issue ?

**Comment by mikemcquaid** [comment]
@sanak Fixed in ac11e0e, thanks!

**Comment by sanak** [comment]
@mikemcquaid
I confirmed that it is fixed.
Thanks, too!
